modules/go-duckdb/1.7.0/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://proxy.golang.org/github.com/marcboeker/go-duckdb/@v/v1.7.0.zip",
```

Comment:
The Go proxy doesn't promise a stable hash, but neither does GitHub for its `/archive` endpoint, so we might as well go with this.

Reply:
I just copied what was in the circl module. TBH, I didn't even realize the go proxy provided a download option like this